src/main/clojure/clara/tools/testing_utils.clj

```diff
@@ -75,3 +75,9 @@
   (f)
   (with-redefs [uc/get-ordered-update-cache ca/get-cancelling-update-cache]
     (f)))
+
```

Comment:
I'd prefer to turn it into a .cljc file rather than create another namespace entirely.  That said I'm OK with addressing the duplication of the join-filter-equals function as part of a later commit rather than as part of these changes.